input/resources/codesystem-calendar-units.xml

```diff
@@ -0,0 +1,125 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+	<id value="calendar-units"/>
+	<url value="http://hl7.org/fhirpath/CodeSystem/calendar-units"/>
+	<version value="5.0.0"/>
+	<name value="CalendarUnits"/>
+	<title value="Calendar Duration Units"/>
+	<status value="active"/>
+	<experimental value="false"/>
+	<date value="2024-02-15"/>
+	<publisher value="HL7 International"/>
+	<contact>
+		<name value="FHIR project team"/>
+		<telecom>
+			<system value="url"/>
+			<value value="http://hl7.org/fhir"/>
+		</telecom>
+	</contact>
+	<description value="Calendar Duration Units - which are disctinct to the UCUM units"/>
+	<caseSensitive value="true"/>
+	<content value="complete"/>
+	<concept>
+		<code value="millisecond"/>
+		<display value="Millisecond"/>
+		<definition value="A calendar millisecond - 1/1000 of a calendar second"/>
+		<designation>
+			<use>
+				<system value="http://snomed.info/sct"/>
+				<code value="900000000000013009"/>
+				<display value="Synonym"/>
+			</use>
+			<value value="Milliseconds"/>
+		</designation>
+	</concept>
+	<concept>
+		<code value="second"/>
+		<display value="Second"/>
+		<definition value="A calendar second"/>
+		<designation>
+			<use>
+				<system value="http://snomed.info/sct"/>
+				<code value="900000000000013009"/>
+				<display value="Synonym"/>
+			</use>
+			<value value="Seconds"/>
+		</designation>
+	</concept>
+	<concept>
+		<code value="minute"/>
+		<display value="Minute"/>
+		<definition value="A calendar minute - excactly 60 seconds"/>
+		<designation>
+			<use>
+				<system value="http://snomed.info/sct"/>
+				<code value="900000000000013009"/>
+				<display value="Synonym"/>
+			</use>
+			<value value="Minutes"/>
+		</designation>
+	</concept>
+	<concept>
+		<code value="hour"/>
+		<display value="Hour"/>
+		<definition value="A calendar hour - exactly 60 minutes"/>
+		<designation>
+			<use>
+				<system value="http://snomed.info/sct"/>
+				<code value="900000000000013009"/>
+				<display value="Synonym"/>
+			</use>
+			<value value="Hours"/>
+		</designation>
+	</concept>
+	<concept>
+		<code value="day"/>
+		<display value="Day"/>
+		<definition value="A calendar day - exactly 24 hours"/>
+		<designation>
+			<use>
+				<system value="http://snomed.info/sct"/>
+				<code value="900000000000013009"/>
+				<display value="Synonym"/>
+			</use>
+			<value value="Days"/>
+		</designation>
+	</concept>
+	<concept>
+		<code value="week"/>
+		<display value="Week"/>
+		<definition value="A calendar week - 7 days exactly"/>
```

Comment:
I think really it is only months are years that are defined differently from UCUM.  At https://ucum.org/ucum I see 1 min = 60 s, 1 h = 60 min, 1 d = 24 , and 1 wk = 7 d

Reply:
There is also some relationship to where fractional seconds come into play, but I don't remember the specifics.
As in a day isn't exactly 24 hours, there are a some extra leap seconds or something whacky, but those are ignored in calendar units.
Happy to take any suggestions on how to change things here.

Reply:
You can see at https://ucum.org/ucum#iso1000 that it ucum a day is exactly 24 hours-- no leap seconds are involved.  I don't think either UCUM or FHIRPath deals with leap seconds.

Reply:
Another great comment, can you submit a tracker to address this change though?

Reply:
I see, this is about the new CodeSystem, and I agree for week and smaller, no need for the calendar modifier

Reply:
Should probably define a complete ValueSet for the CodeSystem too?

Reply:
Done, updated definitions and also added the valueset too